src/lib.cairo

```diff
@@ -28,27 +28,67 @@ struct DataRoot {
     data_root: u256
 }
 
+mod Errors {
+    //Malformed current validator set.
+    const MALFORMED_CURRENT_VALIDATOR_SET: felt252 = 'Malformed current validator set';
+    //Validator signature does not match.
+    const INVALID_SIGNATURE: felt252 = 'Invalid signature';
+    // Submitted validator set signatures do not have enough power.
+    const INSUFFICIENT_VOTING_POWER: felt252 = 'Sub validator inadequate power';
+    // New validator set nonce must be greater than the current nonce.
+    const INVALID_VALIDATOR_SET_NONCE: felt252 = 'New set nonce > current nonce';
+    // Supplied current validators and powers do not match checkpoint.
+    const SUPPLIED_VALIDATOR_SET_INVALID: felt252 = 'Validator&power not matching cp';
```

Comment:
```suggestion
    const SUPPLIED_VALIDATOR_SET_INVALID: felt252 = 'Validator & power not matching cp';
```

Reply:
The issue here is that if I add spaces here, I will get a size 33 and the max is 31.

Reply:
Roger, the way you have it is all good